src/indieweb_utils/feeds/discovery.py

```diff
@@ -67,3 +68,39 @@ def discover_web_page_feeds(url: str, user_mime_types: Optional[List[str]] = Non
         feeds.append(FeedUrl(url=url, mime_type="text/html", title=page_title.text))
 
     return feeds
+
+
+def discover_h_feed(url: str) -> dict:
```

Comment:
Use a proper type, instead of a class.

```suggestion
def discover_h_feed(url: str) -> Dict:
```